schemas/plugins/ProgressPlugin.json

```diff
@@ -0,0 +1,42 @@
+{
+  "definitions": {
+    "HandlerFunction": {
+      "description": "Function that executes for every progress step",
+      "instanceof": "Function",
+      "tsType": "((percentage: number, msg: string, ...args) => void)"
+    }
+  },
+  "title": "ProgressPluginArgument",
+  "oneOf": [
+    {
+      "title": "ProgressPluginOptions",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "handler": {
+          "description": "Function that executes for every progress step",
+          "anyOf": [
+            {
+              "$ref": "#/definitions/HandlerFunction"
+            }
+          ]
+        },
+        "mode": {
+          "description": "Counting mode. Default: modules",
+          "enum": ["modules", "entries"]
+        },
+        "modulesCount": {
+          "description": "Minimum modules count to start with. Only for mode=modules. Default: 500",
+          "type": "number"
+        },
+        "profile": {
+          "description": "Collect profile data for progress steps. Default: false",
+          "enum": [true, false, null]
```

Comment:
`"enum": [true, false, null]` -> `type: "boolean"`

null is not valid

Reply:
webpack-dev-server sets this options to `null` if nothing settled to it - https://github.com/webpack/webpack-dev-server/blob/474022476dc14756f437c1a3f7397f178135453d/bin/webpack-dev-server.js#L348 and it breaks on scheme validation :(

Reply:
hmm ok